src/main/java/com/hpe/simulap/protocol/sip/sampler/SipSampler.java

```diff
@@ -1667,7 +1669,7 @@ private void receiveAnswerMessage(SampleResult res) {
                 SipResponseTransaction aResp = respQueue.poll(timeout, TimeUnit.MILLISECONDS);
                 if ( aResp == null){
                     // No message received
-                	if (_logger.isDebugEnabled()) _logger.debug("incoming response for id "+id+ " not received within timeout "+timeout,null);
+			if (_logger.isDebugEnabled()) _logger.debug("incoming response for id "+id+ " not received within timeout "+timeout,"");
```

Comment:
debug method could be rewritten here like this if (_logger.isDebugEnabled()) _logger.debug("incoming response for id {} not received within timeout {}, id, timeout);